en/08_Changelogs/5.3.0.md

```diff
@@ -51,6 +52,26 @@ Out of the box `silverstripe/assets` provides the new [`InterventionImageFileCon
 
 See [convert a file to a different format](/developer_guides/files/file_manipulation/#file-conversion) for more details.
 
+### Improve customisability of rendered images {#image-rendering}
+
+We've aligned how images added to an HTMLEditorField and SS template images are rendered. The markup up for both `<img>` tag is now generated from the same template file. This will make it easier to customise how images are rendered on your site.
+
+Simply add a `templates/SilverStripe/Assets/Storage/DBFile_Image.ss` file to your Silverstripe CMS theme.
+
+This opens many interesting possibilities. For example, this code snippet will provide a WebP version of any image rendered on your site.
+
+```html
+<picture>
+    <source srcset="{$Convert('webp').Link}" type="image/webp">
```

Comment:
```suggestion
    <source srcset="<% $Convert('webp').Link %>" type="image/webp">
```
Can you use those brackets in templates? I would have thought you'd need the regular template "brackets"

Reply:
```suggestion
    <source srcset="$Convert('webp').Link" type="image/webp">
```
This is more standard.
The suggestion to use `<%` would give you an error.

Reply:
I took Guy's suggestion.